src/bin-utils/index.test.js

```diff
@@ -1,114 +1,113 @@
+/* eslint import/newline-after-import:0, global-require:0 */
 import {resolve} from 'path'
-import test from 'ava'
 import colors from 'colors/safe'
 import {spy} from 'sinon'
-import proxyquire from 'proxyquire'
 import {getScriptsAndArgs, help, preloadModule, loadConfig} from './index'
 
-proxyquire.noCallThru()
-
-test('getScriptsAndArgs: gets scripts', t => {
+test('getScriptsAndArgs: gets scripts', () => {
   const {scripts} = getScriptsAndArgs({
     args: ['boo'],
     rawArgs: ['node', 'p-s', 'boo'],
   })
-  t.deepEqual(scripts, ['boo'])
+  expect(scripts).toEqual(['boo'])
 })
 
-test('getScriptsAndArgs: gets scripts in series', t => {
+test('getScriptsAndArgs: gets scripts in series', () => {
   const {scripts, args} = getScriptsAndArgs({
     args: ['boo,bar'],
     rawArgs: ['node', 'p-s', 'boo,bar'],
   })
-  t.deepEqual(scripts, ['boo', 'bar'])
-  t.deepEqual(args, '')
+  expect(scripts).toEqual(['boo', 'bar'])
+  expect(args).toEqual('')
 })
 
-test('getScriptsAndArgs: gets parallel scripts', t => {
+test('getScriptsAndArgs: gets parallel scripts', () => {
   const {scripts} = getScriptsAndArgs({
     parallel: 'boo,baz',
     rawArgs: ['node', 'p-s', '-p', 'boo,baz'],
   })
-  t.deepEqual(scripts, ['boo', 'baz'])
+  expect(scripts).toEqual(['boo', 'baz'])
 })
 
-test('getScriptsAndArgs: passes args to scripts', t => {
+test('getScriptsAndArgs: passes args to scripts', () => {
   const {args, scripts} = getScriptsAndArgs({
     args: ['boo'],
     rawArgs: ['node', 'p-s', 'boo', '--watch', '--verbose'],
   })
-  t.deepEqual(scripts, ['boo'])
-  t.is(args, '--watch --verbose')
+  expect(scripts).toEqual(['boo'])
+  expect(args).toBe('--watch --verbose')
 })
 
-test('getScriptsAndArgs: returns empty scripts and args if not parallel and no args', t => {
+test('getScriptsAndArgs: returns empty scripts and args if not parallel and no args', () => {
   const {args, scripts} = getScriptsAndArgs({
     args: [],
     rawArgs: ['node', 'p-s'],
   })
-  t.is(scripts.length, 0)
-  t.is(args, '')
+  expect(scripts.length).toBe(0)
+  expect(args).toBe('')
 })
 
-test('preloadModule: resolves a relative path', t => {
-  const relativePath = './fixtures/my-module'
+test('preloadModule: resolves a relative path', () => {
+  // this is relative to process.cwd() I think...
+  // Because of some fancy stuff that Jest does with requires...
+  const relativePath = './src/bin-utils/fixtures/my-module'
```

Comment:
https://github.com/kentcdodds/p-s/blob/master/src/bin-utils/index.js#L83

if you care about what process.cwd gives you, I'd recommend doing `process.cwd = jest.fn(() => '/some/path')`. Your test shouldn't rely on the real value of process.cwd.